python/ray/__init__.py

```diff
@@ -54,6 +54,7 @@
 from ray.worker import (SCRIPT_MODE, WORKER_MODE, LOCAL_MODE,
                         SILENT_MODE)  # noqa: E402
 from ray.worker import global_state  # noqa: E402
+import ray.internal  # noqa: E402
```

Comment:
I don't think this is needed.

Reply:
If I don't add this line, `ray.internal.free` will fail with `AttributeError: module 'ray' has no attribute 'internal'`.